recipes/erlang/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+export LIBRARY_PATH="${PREFIX}/lib:${LIBRARY_PATH}"
+export LD_LIBRARY_PATH="${PREFIX}/lib:${LD_LIBRARY_PATH}"
+export ERL_TOP="$(pwd)"
+./configure --with-ssl="${PREFIX}" --prefix="${PREFIX}" --without-javac
```

Comment:
So, I did some more investigation as I know parallelism is super important to Erlang and we want to make sure it is still usable for real applications. Atomic memory operations are very important to Erlang (see [Atomic Memory Operations and the VM]( http://erlang.org/doc/installation_guide/INSTALL.html#id59102 )). We need to provide this if a system doesn't have native atomics. So, we should really consider it. That being said there appears to be at least one recent unsolved [bug]( http://bugs.erlang.org/browse/ERL-78 ) with using `libatomic_ops`. Also, according to the [message]( https://patchwork.ozlabs.org/patch/572040/ ) in this patch native atomics are preferred for 18.x, which is what we have here. So, let's skip `libatomic_ops` after all and see if anyone complains later or asks for 17.x or lower.

The 32-bit/64-bit stuff is easy to do, but also not leveraged by us currently. So, I'm not too worried about it.